packages/cli/src/lib/webdriver.ts

```diff
@@ -17,7 +17,7 @@ const startDriver = async (
 
     let options = new chrome.Options();
     // selenium-webdriver < 4.17.0
-    if (typeof options.headless === 'function') {
+    if ('headless' in options && typeof options.headless === 'function') {
```

Comment:
Newer version of webdriverio seems to have removed the headless function from the chrome option type